### DISCUSSION TITLE
Duplicating an item maintains creation date of original note, rather than using time of duplication

### DISCUSSION THREAD

**Discussion by IceFire23** [standardnotes/forum#2466] [Suggestions]
**Describe the bug**
When using the "Action Bar" to "Duplicate" a note, the "Date Added" of the duplicated note is copied from the original note. I expected the "Date Added" would be set to the day/time the note was duplicated.

**To Reproduce**

0. Ensure "Action Bar" extension is installed, and that your notes are sorted using the "Date Added" method
1. Click on a note. Record the "Date added" for that note.
2. Open up the "Action Bar" extension, and then click "Duplicate"
3. Look at the duplicated note in your note list. Compare the "Date added" for the duplicate and note that it is identical to the date/time recorded in step 1.

**Expected behavior**
I expected the duplicated note would have the "Date added" be set to the time the note was duplicated, as that's the date the duplicate note was created.

**Screenshots**
Note how all 3 of the below notes have the same "Date added" field, despite 2 of them being duplicated from the other one at later times. (Original note was the one the one with 1"1:48pm creation" in the title, other two were duplicated from that at the times listed in title)
![DuplicateItem_WrongCreationTime](https://user-images.githubusercontent.com/35338/88466218-e7e58000-ce7e-11ea-965d-5d5d6885fc6f.jpg)

**Desktop (please complete the following information):**
 - OS: Windows 10 v1909

**Additional Info:**
This affects my workflow because when all the notes have the same "Date added" time, it messes up the ordering when I sort by that field.

**Comment by JaspalSuri** [comment]
Hi IceFire23, thanks for the feedback! I will discuss this with the team and get back to you.

**Comment by atmoio** [comment]
Hmm, I believe the original behavior was as you described, but was modified later to be the present behavior. But thinking about it now, using the time of duplication makes more sense. The issue is that since the default scroll order is "Date Added", if you are deep in your list of notes and choose to duplicate, the note will be duplicated, but you won't see any changes in the UI as the new note will now be sorted at the top. So the solution here involves not just a quick code change to change the date, but also a fundamental UX change so that there's some sort of interaction confirmation the duplication, or taking you to the duplicated note (while somehow letting you know that that's what happened).

**Comment by JaspalSuri** [comment]
Would it be possible to add a tag to notes that are duplicated so that they can be easily found later? It's still possible that some users might not notice the tag or it could be annoying to have to remove it each time, though.

**Comment by atmoio** [comment]
Hmm not a bad idea. I think in general if we can have a conspicuous animation for transitioning from one note to another it might solve this. But currently our transitions are immediate so you wouldn't even notice that the active note was changed since they have exactly the same content.

cc @zsoltszilvai

**Comment by IceFire23** [comment]
Thanks for looking into it!
